.ado/templates/prepare-env.yml

```diff
@@ -28,6 +28,16 @@ steps:
     inputs:
       versionSpec: ">=5.8.0"
 
+  - task: NodeTool@0
+    displayName: Installing Node
+    inputs:
+      versionSpec: '14.x' 
+
+  - task: CmdLine@2
+    displayName: Installing Yarm
+    inputs:
+      script: npm install -g yarn
+
```

Comment:
Wondering why we need this vs using the CI built-in version, or is thin in prep for moving to a separate pool?

Reply:
Oh, I see the windevbuildagents part now

Reply:
Yup, there is some refactoring that needs to happen between setup and sharing of log steps between the various things we do.